client/signup/steps/site-topic/index.jsx

```diff
@@ -18,7 +18,7 @@ import FormLabel from 'components/forms/form-label';
 import InfoPopover from 'components/info-popover';
 import FormFieldset from 'components/forms/form-fieldset';
 import SuggestionSearch from 'components/suggestion-search';
-import { submitSiteTopic } from 'state/signup/steps/site-topic/actions';
+import { submitSiteTopic, setSiteTopic } from 'state/signup/steps/site-topic/actions';
```

Comment:
This is just restoring something that was borked by c350838dd43ef5138ae51323721be19a0bb42501

@southp I'm interested in how you managed to get that commit through the linter - I guess it was just a merge without a rebase?

Reply:
Hey, sorry that I missed your inquiry. To be honest, I'm not 100% sure. I always rebase before merging anything, but that would have been caught by the linter if I had done it.  Thus I think I must have forgotten to rebase that time :/